pkg/apis/deployment/v1/topology_status.go

```diff
@@ -103,6 +103,28 @@ func (t *TopologyStatus) IsTopologyOwned(m *TopologyMemberStatus) bool {
 	return t.ID == m.ID
 }
 
+func (t *TopologyStatus) IsTopologyEvenlyDistributed(group ServerGroup) bool {
```

Comment:
I would like to have a description here. What function should return when `t.Zones` is empty. Currently, 
the function `IsTopologyEvenlyDistributed` returns false (not distributed evenly) when t.Zones are empty. Is it intentional? The description of the function should explain it because I don't know the intensions

Reply:
And t.Zones cant be with size 0 - it is checked in code. If it is, then false is expected